src/rgw/rgw_admin.cc

```diff
@@ -489,6 +489,9 @@ void usage()
   cout << "\nradoslist options:\n";
   cout << "   --rgw-obj-fs                  the field separator that will separate the rados object name from the rgw object name;\n";
   cout << "                                 additionally rados objects for incomplete multipart uploads will not be output\n";
+  cout << "\nBucket list objects options:\n";
+  cout << "   --max-entries                 max number of entries listed, default 1000, absolute maximum 10000\n";
```

Comment:
So I'm asking that you remove `", absolute maximum 10000"`, so the string will then be:

`" --max-entries max number of entries listed\n";`

Reply:
OK, I will remove it